distribution/packages/build.gradle

```diff
@@ -134,9 +134,9 @@ Closure commonPackageConfig(String type, boolean jdk, String architecture) {
     destinationDirectory = file("${prefix}/build/distributions")
 
     // SystemPackagingTask overrides default archive task convention mappings, but doesn't provide a setter so we have to override the convention mapping itself
-    // Deb convention uses a '-' for final separator before architecture, rpm uses a '.'
+    // Deb convention uses a '_' for final separator before architecture, rpm uses a '.'
```

Comment:
@saratvemulapalli do you think it is OK to change '-' to '_'? thanks

Reply:
We should be good. Shouldn't have any issues with DistributionDownloader either.